### DISCUSSION TITLE
Welcome to the OpenEVSE ESP32 WiFi Firmware Discussions!

### DISCUSSION THREAD

**Discussion by chris1howell** [OpenEVSE/openevse_esp32_firmware#619] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zerodur12** [comment]
how to select or configure  the interface langage in the new interface openevse-gui-v2 I haven't been able to find how to do it?

**Reply by zerodur12** [reply]
Find in the HTTP Configuration menu

**Comment by scbiker2** [comment]
For the V5.1.0 release, there's not a pre-compiled bin file for the Huzzah ESP32.. is that intentional and any different instructions for compiling / uploading with the latest builds that allow OTA updates?  Thanks!

**Comment by chris1howell** [comment]
Firmware 5.1.0 only added support for the TFT LCD. There is no difference
for other modules..

**Reply by scbiker2** [reply]
Thanks for clarifying.
